_schemaV2.graphql

```diff
@@ -5698,33 +5698,41 @@ type Mutation {
   # Create a credit card
   createCreditCard(input: CreditCardInput!): CreditCardPayload
 
+  # Attach an gemini asset to a consignment submission
+  createGeminiEntryForAsset(
```

Comment:
There seems to be some unintentional schema updates here (possible not using consignments stitching locally?)

Reply:
This seems to have been caused by the order change in the v2 schema here: https://github.com/artsy/metaphysics/commit/53141bed2e0b4114002f2514d111d2f579231831?diff=unified#diff-6773300ece2ed026b37737f3d57d39bd

I reverted the order change and this unexpected schema change is gone now.